autoPyTorch/evaluation/tae.py

```diff
@@ -207,7 +221,7 @@ def run_wrapper(
         else:
             if run_info.budget == 0:
                 run_info = run_info._replace(budget=self.pipeline_config[self.budget_type])
-            elif run_info.budget <= 0 or run_info.budget > 100:
+            elif run_info.budget <= 0:
                 raise ValueError('Illegal value for budget, must be >0 and <=100, but is %f' %
                                  run_info.budget)
```

Comment:
Incompatible

Reply:
Good point, we should also refactor the error message.